doc/src/compute_count_type.rst

```diff
@@ -0,0 +1,88 @@
+.. index:: compute count/type
+
+compute count/type command
+====================
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   compute ID group-ID count/type mode
+
+* ID, group-ID are documented in :doc:`compute <compute>` command
+* count/type = style name of this compute command
+* mode = {atom} or {bond}
+  
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   compute 1 all count/type atom
+   compute 1 flowmols count/type bond
+
+Description
+"""""""""""
+
+Define a computation that counts the current number of atoms by atom
+type or the number of bonds by bond type.  The latter can be useful in
+reactive simulations where bonds are broken or created.
+
+Note that for this command, bonds are the topological ones enumerated
+in a data file, initially read by the :doc:`read_data <read_data>`
+command.  They do not refer to bonds defined on-the-fly by bond-order
+or reactive pair styles.
+
+These commands can create and break toplogical bonds:
+
+* :doc:`fix bond/react <fix_bond_react>`
+* :doc:`fix bond/create <fix_bond_create>`
+* :doc:`fix bond/break <fix_bond_break>`
+* :doc:`bond_style quartic <bond_quartic>`
+* :doc:`BPM package <Howto_bpm>` bond styles
```

Comment:
I'm pretty sure fix bond/break deletes bonds by shifting bond entries over instead of setting the bond type to zero, so computing the number of broken bonds won't work with that fix. I want to say fix bond/react works the same way, but I'd have to check. 

The text in Howto broken bonds might need to be clarified to emphasize this point too.